presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestEliminateCrossJoins.java

```diff
@@ -65,16 +64,14 @@ public void testEliminateCrossJoin()
                 .setSystemProperty(REORDER_JOINS, "true")
                 .on(crossJoinAndJoin(INNER))
                 .matches(
-                        project(
+                        join(INNER,
```

Comment:
It's required by the change to EliminateCrossJoins.  I'll pull those two files out into a separate commit.